### PR TITLE
General improvement to CLI reference for Teleport daemon and more

### LINKED ISSUE #15592: CLI Documentation is Missing Information
## Details
Our CLI documentation is missing some information and should be audited and revisited. Subcommands of `teleport` are not mentioned at all and we only really touch on options used with `teleport start`.  We can additionally aim to use more descriptive language, such as to make it clear that `--roles` are an option used with `teleport start` and not `teleport` directly. There may be more that I'm not catching, but wanted to make sure this was on our radar to prioritize as needed before performing a larger audit.

### Category
- Improve Existing
- Remove Outdated

### PR DESCRIPTION
**Pull request by sagesyr**
- Adjusts language around `teleport` to better reflect it's status as a background process and not a systemd init daemon.
- Expands `teleport` CLI documentation to reflect new features, better positions `role` configurations where appropriate in an effort to avoid potential confusion.
- Adds a table to `teleport` section in an attempt to lay out the same information more intuitively and succintly, keeps and adds individual sections for subcommands only with a higher level of configurability.

Resolves #15592

### CODE REVIEW

File: docs/pages/reference/cli.mdx [hunk 1]
```diff
@@ -16,31 +16,35 @@ Teleport is made up of three CLI tools.
...
-| [Proxy](../architecture/proxy.mdx) | `proxy` | The gateway that clients use to connect to the Auth Service or resources managed by Teleport. |
-| [App](../application-access/introduction.mdx) | `app` | Runs a daemon on a host that provides access to applications using an SSH reverse tunnel. |
-| [Kube](../kubernetes-access/introduction.mdx) | `kube` | The Teleport daemon will run the Kubernetes Service. |
-| [DB](../database-access/reference.mdx) | `db` | The Teleport daemon will run the Database Service. |
-| [Trusted Cluster](../management/admin/trustedclusters.mdx) | `trusted_cluster` | The Teleport daemon will support a leaf cluster used to connect to another Teleport cluster. |
-| [Windows Desktop](../desktop-access/getting-started.mdx) | `windowsdesktop` | The Teleport daemon will run the Windows Desktop Service. |
-
 ### teleport start
 
+The `teleport start` command includes a large number of optional configuration flags which change the behavior of the default command to support a wider spectrum of teleport products and services.
```
Comment (strideynet) [inline docs/pages/reference/cli.mdx:46]: This might be a good place to guide people towards *not* using those configuration flags and using the YAML config files unless they are experimenting with Teleport features.

File: docs/pages/reference/cli.mdx [hunk 2]
```diff
@@ -6,8 +6,8 @@ h1: Command Line (CLI) Reference
 
 Teleport is made up of three CLI tools.
 
-- [teleport](#teleport): The daemon that runs the Auth Service, Proxy Service, and other Teleport services.
-- [tsh](#tsh): A tool that lets end users interact with Teleport.
+- [teleport](#teleport): A CLI tool that supports the Teleport access plane.
+- [tsh](#tsh): A CLI tool that allows end users to interact with Teleport sessions and list cluster information.
 - [tctl](#tctl): An administrative tool that can configure the Teleport Auth Service.
```
Comment (strideynet) [inline docs/pages/reference/cli.mdx:11]: If the other two are described as a CLI tool, `tctl` certainly should be as well.

File: docs/pages/reference/cli.mdx [hunk 3]
```diff
@@ -6,8 +6,8 @@ h1: Command Line (CLI) Reference
 
 Teleport is made up of three CLI tools.
 
-- [teleport](#teleport): The daemon that runs the Auth Service, Proxy Service, and other Teleport services.
-- [tsh](#tsh): A tool that lets end users interact with Teleport.
+- [teleport](#teleport): A CLI tool that supports the Teleport access plane.
```
Comment (strideynet) [inline docs/pages/reference/cli.mdx:9]: It's a little odd to hear `teleport` described as a CLI tool, but I suppose it's not entirely wrong to say so. `daemon` was definitely more misleading as Teleport runs as a foreground process. So perhaps this will do for now.

File: docs/pages/reference/cli.mdx [hunk 4]
```diff
@@ -67,6 +71,29 @@ clusters.
...
 | `--app-public-addr` | none | none | Public address fo the application to proxy |
 
+#### teleport start --roles
+
+The `--roles` flag when used with `teleport --start` instructs Teleport on which specific Teleport services to start. Below is a more cohesive table of roles and their associated services that `teleport start` supports:
+
+| Service | Role Name | Description |
+| - | - | - |
+| [Node](../architecture/nodes.mdx) | `node` | Runs a daemon on a host that allows SSH connections from authenticated clients. |
+| [Auth](../architecture/authentication.mdx) | `auth` | Authenticates hosts and users who want access to Teleport-managed resources or information about a cluster. |
```
Comment (GavinFrazar) [inline docs/pages/reference/cli.mdx:81]:
```suggestion
| [Auth](../architecture/authentication.mdx) | `auth` | Authenticates and authorizes hosts and users who want access to Teleport-managed resources or information about a cluster. |
```

File: docs/pages/reference/cli.mdx [hunk 5]
```diff
@@ -67,6 +71,29 @@ clusters.
...
+| [Proxy](../architecture/proxy.mdx) | `proxy` | The gateway that clients use to connect to the Auth Service or resources managed by Teleport. |
+| [App](../application-access/introduction.mdx) | `app` | Runs a daemon on a host that provides access to applications using an SSH reverse tunnel. |
```
Comment (GavinFrazar) [inline docs/pages/reference/cli.mdx:83]:
```suggestion
| [App](../application-access/introduction.mdx) | `app` | Provides access to applications. |
```

Remove "daemon", and I'm not sure the reverse tunnel is relevant info - kind of an implementation detail, although if you want to emphasize the consequence of that detail (Teleport only requires the proxy service to be externally reachable thanks to reverse tunnels) then that would be useful info.

File: docs/pages/reference/cli.mdx [hunk 6]
```diff
@@ -67,6 +71,29 @@ clusters.
...
+| [Kube](../kubernetes-access/introduction.mdx) | `kube` | The Teleport daemon will run the Kubernetes Service. |
```
Comment (GavinFrazar) [inline docs/pages/reference/cli.mdx:84]:
```suggestion
| [Kube](../kubernetes-access/introduction.mdx) | `kube` | Provides access to Kuberenetes clusters. |
```

"daemon" here

File: docs/pages/reference/cli.mdx [hunk 7]
```diff
@@ -67,6 +71,29 @@ clusters.
...
+| [DB](../database-access/reference.mdx) | `db` | The Teleport daemon will run the Database Service. |
```
Comment (GavinFrazar) [inline docs/pages/reference/cli.mdx:85]:
```suggestion
| [DB](../database-access/reference.mdx) | `db` | Provides access to databases. |
```

"daemon" here

File: docs/pages/reference/cli.mdx [hunk 8]
```diff
@@ -67,6 +71,29 @@ clusters.
...
+| [Trusted Cluster](../management/admin/trustedclusters.mdx) | `trusted_cluster` | The Teleport daemon will support a leaf cluster used to connect to another Teleport cluster. |
```
Comment (GavinFrazar) [inline docs/pages/reference/cli.mdx:86]: Is `trusted_cluster` a role now? I don't see it in the master branch lib/config/configuration.go:validateRoles func.

File: docs/pages/reference/cli.mdx [hunk 9]
```diff
@@ -67,6 +71,29 @@ clusters.
...
+| [Windows Desktop](../desktop-access/getting-started.mdx) | `windowsdesktop` | The Teleport daemon will run the Windows Desktop Service. |
```
Comment (GavinFrazar) [inline docs/pages/reference/cli.mdx:87]:
```suggestion
| [Windows Desktop](../desktop-access/getting-started.mdx) | `windowsdesktop` | Provides Windows desktop access. |
```

"daemon" here

File: docs/pages/reference/cli.mdx [hunk 10]
```diff
@@ -6,8 +6,8 @@ h1: Command Line (CLI) Reference
 
 Teleport is made up of three CLI tools.
```
Comment (GavinFrazar) [inline docs/pages/reference/cli.mdx:7]: I know this in the original doc, but why don't we mention `tbot`?

File: docs/pages/reference/cli.mdx (hunk 3 above)
Comment (rsyracuse) [reply docs/pages/reference/cli.mdx:9]: I struggled with the wording on this as well, but felt that it was best to lean into CLI ability given the context of the doc. Maybe it'd be better to say "A CLI tool and foreground process..." just to be more accurate?

File: docs/pages/reference/cli.mdx [hunk 11]
```diff
@@ -4,11 +4,12 @@ description: Detailed guide and reference documentation for Teleport's command l
...
 
-Teleport is made up of three CLI tools.
+Teleport is made up of four CLI tools.
 
-- [teleport](#teleport): A CLI tool that supports the Teleport access plane.
+- [teleport](#teleport): A process and CLI tool that supports the Teleport access plane.
 - [tsh](#tsh): A CLI tool that allows end users to interact with Teleport sessions and list cluster information.
-- [tctl](#tctl): An administrative tool that can configure the Teleport Auth Service.
+- [tctl](#tctl): An administrative CLI tool that can configure the Teleport Auth Service.
+- [tbot](#tbot): A CLI tool that supports Machine ID on Teleport, used to apply short lvied certificates to service accounts (e.g, a CI/CD server).
```
Comment (GavinFrazar) [inline docs/pages/reference/cli.mdx:12]:
```suggestion
- [tbot](#tbot): A CLI tool that supports Machine ID on Teleport, used to apply short lived certificates to service accounts (e.g, a CI/CD server).
```

File: docs/pages/reference/cli.mdx [hunk 12]
```diff
@@ -2013,3 +2013,221 @@ $ tsh ls --search=staging,mac
...
+$ tbot init \
+    --destination-dir=/opt/machine-id \
+    --bot-user=teleport \
+    --reader-user=jenkins
+```
+
+
+### tbot db
+
+Allows connections to databases using native clients and queries database information. This is best used for testing and validation purposes;
```
Comment (GavinFrazar) [inline docs/pages/reference/cli.mdx:2134]:
```suggestion
Connects to databases using native clients and queries database information. This is best used for testing and validation purposes;
```

nit: I would rephrase like this, since `tbot db` isn't allowing connections, it's just connecting

**Review by GavinFrazar — APPROVED** [review]
left a couple comments but LGTM

**Review by strideynet — APPROVED** [review]
Looks good once comments by Gavin have been addressed.

File: docs/pages/reference/cli.mdx [hunk 13]
```diff
@@ -4,50 +4,56 @@ description: Detailed guide and reference documentation for Teleport's command l
...
-more "roles" with the `--roles` flags. The arguments to `--roles` correspond to
-the following services.
-
-<Notice type="tip" scope={["cloud"]}>
-
-Teleport Cloud manages Teleport instances with the `auth` and `proxy` roles. Use
-the remaining roles to manage access to specific resources and other Teleport
-clusters.
-
+The CLI tool that supports the Teleport access plane is called `teleport`, and allows Teleport services to be created, configured, and otherwise interacted with over the command line.
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:20]: - Same comment as above re: "supports the Teleport access plane"
- What does it mean for a Teleport service to be "created"? Should we say instead that it runs Teleport services, and list the services that it runs?

File: docs/pages/reference/cli.mdx [hunk 14]
```diff
@@ -4,50 +4,56 @@ description: Detailed guide and reference documentation for Teleport's command l
...
+
+| Command | Description |
+| - | - |
+| `teleport help` | Outputs guidance for using commands on Teleport. |
+| `teleport start` | Starts the `teleport` service using the current shell session. This command will not start the `teleport` service as a daemon. |
+| `teleport status` | Prints the status of the current active Teleport SSH session. |
+| `teleport configure` | Generates and writes a [configuration YAML file](config.mdx) for the Teleport service. This file should be customized in production to suit the needs of your environment, and the default output should only be used when testing. |
+| `teleport version` | Prints the current release version of the Teleport binary installed on your system. |
+| `teleport app start` | Starts the application proxy service. |
+| `teleport db start` | Starts the database proxy service. |
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:33]:
```suggestion
| `teleport db start` | Starts the Teleport Database Service. |
```

File: docs/pages/reference/cli.mdx [hunk 15]
```diff
@@ -4,50 +4,56 @@ description: Detailed guide and reference documentation for Teleport's command l
...
+| `teleport db configure create` | Generates a database service configuration. |
+| `teleport db configure bootstrap` | Used to bootstrap a configuration to the Teleport database agent by reading a provided configuration. |
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:35]:
```suggestion
| `teleport db configure bootstrap` | Used to bootstrap a configuration to the Teleport Database Service by reading a provided configuration. |
```

File: docs/pages/reference/cli.mdx [hunk 16]
```diff
@@ -4,50 +4,56 @@ description: Detailed guide and reference documentation for Teleport's command l
...
+| `teleport db configure aws print-iam` | Generates and outputs current IAM policies for a teleport managed database. |
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:36]:
```suggestion
| `teleport db configure aws print-iam` | Generates and outputs current IAM policies for a Teleport-managed database. |
```

File: docs/pages/reference/cli.mdx [hunk 17]
```diff
@@ -4,50 +4,56 @@ description: Detailed guide and reference documentation for Teleport's command l
...
+| `teleport db configure aws create-iam` | Generates, creates, and attaches desired IAM policies to a teleport managed database. |
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:37]:
```suggestion
| `teleport db configure aws create-iam` | Generates, creates, and attaches the desired IAM policies to a Teleport-managed database. |
```

File: docs/pages/reference/cli.mdx [hunk 18]
```diff
@@ -4,50 +4,56 @@ description: Detailed guide and reference documentation for Teleport's command l
...
+| `teleport install systemd` | Creates a systemd unit file, used to configure and install a `teleport` service daemon. |
+| `teleport node configure` | Generates a configuration file for a Teleport node accessed via SSH. |
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:39]:
```suggestion
| `teleport node configure` | Generates a configuration file for a Teleport Node accessed via SSH. |
```
To distinguish this from the general term "node" (https://goteleport.com/docs/architecture/overview/#definitions).

File: docs/pages/reference/cli.mdx [hunk 19]
```diff
@@ -4,50 +4,56 @@ description: Detailed guide and reference documentation for Teleport's command l
...
+
+<Notice type="tip">
+For more information on subcommands when working with the `teleport` cli, use the `--help` option, or `teleport <subcommand> --help`.
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:42]:
```suggestion
For more information on subcommands when working with the `teleport` CLI, use the `--help` option or `teleport <subcommand> --help`.
```

**Review by ptgott — COMMENTED** [review]
I've reviewed part of this and left comments. I still need to review the rest, and I've blocked off time next week for it. Sorry for the long wait!

File: docs/pages/reference/cli.mdx [hunk 20]
```diff
@@ -2034,3 +2028,221 @@ $ tsh ls --search=staging,mac
...
+
+Additionally, be aware of the following limitations of `tbot db`:
+ - `tbot db connect` requires a `tbot db login` for certain database types,
+   like MySQL, so that additional connection parameters can be written to a
+   local configuration file.
+ - `tbot db env` is not fully supported.
+
+### tbot proxy
+
+Allows for access to teleport resources via a proxy using various protocols. Requires running a local ALPN/SNI proxy to access the resources if using TLS.
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:2181]:
```suggestion
Allows for access to Teleport resources via a proxy using various protocols. Requires running a local ALPN/SNI proxy to access the resources if using TLS.
```

Comment (ptgott) [reply docs/pages/reference/cli.mdx:2181]: It's difficult for me to tell what this command does from this paragraph. 

- What does "via a proxy" mean? What kind of proxy is this? How would a user know whether a given proxy is relevant to this command?

- How do you know whether you have the required local ALPN/SNI proxy running? Also, `tsh proxy db` starts the local ALPN/SNI proxy for you—does this command do that as well?

File: docs/pages/reference/cli.mdx [hunk 21]
```diff
@@ -2034,3 +2028,221 @@ $ tsh ls --search=staging,mac
...
+The `tbot proxy` command acts as a wrapper for `tsh proxy` to provide local proxy functionality for various protocols.
+
+Note that tsh must be installed to make use of this command.
+
+#### Flags
+
+| Flag                | Description                                                                                              |
+|---------------------|----------------------------------------------------------------------------------------------------------|
+| `-d/--debug`        | Enable verbose logging to stderr.                                                                        |
+| `-c/--config`       | Path to a configuration file.                                                                            |
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:2191]: Is this required? If not, what's the default?

File: docs/pages/reference/cli.mdx [hunk 22]
```diff
@@ -1,53 +1,68 @@
...
-h1: Command Line (CLI) Reference
 ---
 
-Teleport is made up of three CLI tools.
+Teleport is made up of four CLI tools.
 
-- [teleport](#teleport): The daemon that runs the Auth Service, Proxy Service, and other Teleport services.
-- [tsh](#tsh): A tool that lets end users interact with Teleport.
-- [tctl](#tctl): An administrative tool that can configure the Teleport Auth Service.
+- [teleport](#teleport): A CLI tool that supports the Teleport access plane by starting and configuring various Teleport services.
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:8]: I don't think we need to repeat "CLI tool" in each list item, since we already say that each list item describes a CLI tool.

File: docs/pages/reference/cli.mdx [hunk 23]
```diff
@@ -1,53 +1,68 @@
...
-<Notice type="tip" scope={["cloud"]}>
-
-Teleport Cloud manages Teleport instances with the `auth` and `proxy` roles. Use
-the remaining roles to manage access to specific resources and other Teleport
-clusters.
-
+The CLI tool that supports the Teleport access plane is called `teleport`, and allows Teleport services to be managed
+over the command line:
+
+- [Auth](../architecture/authentication.mdx)
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:22]: I know it would repeat "Service", but the full name of each service is the "Auth Service", "Proxy Service", etc. Otherwise, I'm not sure it's clear what a user should expect each link in this list to point to.

File: docs/pages/reference/cli.mdx [hunk 24]
```diff
@@ -1,53 +1,68 @@
...
+- [Node/SSH](../architecture/nodes.mdx)
+- [Proxy](../architecture/proxy.mdx)
+- [App](../application-access/introduction.mdx)
+- [Database](../database-access/introduction.mdx)
+- [Windows Desktop](../desktop-access/introduction.mdx)
+- [Kubernetes](../kubernetes-access/introduction.mdx)
+- [Generate a systemd unit file for a systemd managed service daemon](../management/admin/daemon.mdx).
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:29]: All of the items in this list are links to documentation about a Teleport service except for this link and the one below. Why are these links here?

File: docs/pages/reference/cli.mdx [hunk 25]
```diff
@@ -1,53 +1,68 @@
...
+The primary commands for the `teleport` CLI are as follows:
+
+| Command | Description |
+| - | - |
+| `teleport help` | Outputs guidance for using Teleport commands. |
+| `teleport start` | Starts the `teleport` process in the foregound using the current shell session, including any services configured by the [configuration YAML file](config.mdx). |
+| `teleport status` | Prints the status of the current active Teleport SSH session. |
+| `teleport configure` | Generates and writes a [configuration YAML file](config.mdx) for the Teleport service. This file should be customized in production to suit the needs of your environment, and the default output should only be used when testing. |
+| `teleport version` | Prints the current release version of the Teleport binary installed on your system. |
+| `teleport app start` | Starts the application proxy service. |
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:41]:
```suggestion
| `teleport app start` | Starts the Teleport Application Service. |
```

File: docs/pages/reference/cli.mdx [hunk 26]
```diff
@@ -1,53 +1,68 @@
...
-| [Proxy](../architecture/proxy.mdx) | `proxy` | The gateway that clients use to connect to the Auth Service or resources managed by Teleport. |
-| [App](../application-access/introduction.mdx) | `app` | Runs a daemon on a host that provides access to applications using an SSH reverse tunnel. |
-| [Kube](../kubernetes-access/introduction.mdx) | `kube` | The Teleport daemon will run the Kubernetes Service. |
-| [DB](../database-access/reference.mdx) | `db` | The Teleport daemon will run the Database Service. |
-| [Trusted Cluster](../management/admin/trustedclusters.mdx) | `trusted_cluster` | The Teleport daemon will support a leaf cluster used to connect to another Teleport cluster. |
-| [Windows Desktop](../desktop-access/getting-started.mdx) | `windowsdesktop` | The Teleport daemon will run the Windows Desktop Service. |
-
 ### teleport start
 
+The `teleport start` command includes a large number of optional configuration flags which change the behavior of the default command to support a wider spectrum of Teleport services.
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:56]:
```suggestion
The `teleport start` command includes a large number of optional configuration flags.
```
I think we can make this sentence much more concise. Do we need more than this part of it? For me, the crux of the paragraph is that `teleport start` is unwieldy and using a configuration file is preferred.

File: docs/pages/reference/cli.mdx [hunk 27]
```diff
@@ -2072,3 +2060,248 @@ $ tsh ls --search=staging,mac
...
+## tbot
+
+`tbot` is a CLI tool used with **Machine ID** that programatically issues and renews
+short-lived certificates to any service account (e.g, a CI/CD server).
+
+The primary commands for `tbot` are as follows:
+
+| Command | Description |
+| - | - |
+| `tbot help` | Outputs guidance for using commands with `tbot` |
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:2073]:
```suggestion
| `tbot help` | Outputs guidance for using commands with `tbot`. |
```
So punctuation is consistent in this column

File: docs/pages/reference/cli.mdx [hunk 28]
```diff
@@ -2072,3 +2060,248 @@ $ tsh ls --search=staging,mac
...
+| `tbot version` | Outputs the current version of the `tbot` binary. |
+| `tbot configure` | Outputs a basic Machine ID client configuration file to be adjusted as needed. |
+| `tbot start` | Starts the Machine ID client `tbot`, fetching and writing certificates to disk at a set interval. |
+| `tbot init` | Initialize a certificate destination directory for writes from a separate bot user, configuring either file or POSIX ACL permissions. |
+| `tbot db` | Connects to databases using native clients and queries database information. Functions as a wrapper for `tsh`, and requires `tsh` installation. |
+| `tbot proxy` | Allows for access to Teleport resources on a cluster using TLS routing. Functions as a wrapper for `tsh`, and requires `tsh` installation. |
+
+### tbot start
+
+Starts the Machine ID client tbot, fetching and writing certificates to disk at a set interval.
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:2083]:
```suggestion
Starts the Machine ID client `tbot`, fetching and writing certificates to disk at a set interval.
```

File: docs/pages/reference/cli.mdx [hunk 29]
```diff
@@ -2072,3 +2060,248 @@ $ tsh ls --search=staging,mac
...
+| `--data-dir`         | Directory to store internal bot data. In production environments access to this directory should be limited only to an isolated linux user as an owner with `0600` permissions. |
+| `--destination-dir`  | Directory to write short-lived machine certificates.                                           |
+| `--certificate-ttl`  | TTL of short-lived machine certificates.                                                       |
+| `--renewal-interval` | Interval at which short-lived certificates are renewed; must be less than the certificate TTL. |
+| `--join-method`      | Method to use to join the cluster. Can be `token` or `iam`.                                    |
+| `--oneshot`          | If set, quit after the first renewal.                                                          |
+
+#### Examples
+
+<ScopedBlock scope={["cloud"]}>
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:2103]: I think we should use a `Tabs` component instead in case a user has forgotten to select a scope (which is most of the time).

File: docs/pages/reference/cli.mdx [hunk 30]
```diff
@@ -2072,3 +2060,248 @@ $ tsh ls --search=staging,mac
...
+   --join-method=token \
+   --ca-pin=sha256:1111111111111111111111111111111111111111111111111111111111111111 \
+   --auth-server=auth.example.com:3025
+```
+
+</ScopedBlock>
+
+### tbot init
+
+Allows certificates to be written to disks other than a Machine ID client, configuring either file or POSIX ACL permissions.
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:2133]: What does `tbot init` do? This sentence just describes its effects.

File: docs/pages/reference/cli.mdx [hunk 31]
```diff
@@ -2072,3 +2060,248 @@ $ tsh ls --search=staging,mac
...
+| `--bot-user`        | Enables POSIX ACLs and defines the Linux user that can read/write short-lived certificates to `--destination-dir`. |
+| `--reader-user`     | Enables POSIX ACLs and defines the Linux user that will read short-lived certificates from `--destination-dir`.    |
+| `--init-dir`        | If using a config file and multiple destinations are configured, controls which destination dir to configure.      |
+| `--clean`           | If set, remove unexpected files and directories from the destination.                                              |
+
+#### Examples
+
+**Example using file permissions.**
+
+The following command highlights how to set permissions with tbot through Linux groups, using the user and group `jenkins:jenkins`.
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:2153]:
```suggestion
The following command highlights how to set permissions with `tbot` through Linux groups, using the user and group `jenkins:jenkins`.
```

File: docs/pages/reference/cli.mdx [hunk 32]
```diff
@@ -2072,3 +2060,248 @@ $ tsh ls --search=staging,mac
...
+   like MySQL, so that additional connection parameters can be written to a
+   local configuration file.
+ - `tbot db env` is not fully supported.
+
+### tbot proxy
+
+Allows for access to Teleport resources via a local TLS proxy in single-port mode.
+The `tbot proxy` command acts as a wrapper for `tsh proxy` to provide local proxy functionality for various protocols.
+
+Note that tsh must be installed to make use of this command.
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:2216]:
```suggestion
Note that `tsh` must be installed to make use of this command.
```

File: docs/pages/reference/cli.mdx (hunk 21 above)
Comment (ptgott) [reply docs/pages/reference/cli.mdx:2223]: I'm still not sure what the default value for this is. Also, is this a `tbot` configuration profile or a `tsh` configuration file?

File: docs/pages/reference/cli.mdx [hunk 33]
```diff
@@ -2072,3 +2060,248 @@ $ tsh ls --search=staging,mac
...
+The following command forwards standard input and output over a proxy suitable for use as an OpenSSH `ProxyCommand` for SSH access:
+
+```code
+$ tbot proxy --destination-dir=./tbot-user --proxy=proxy.example.com:3080 ssh alice@node:3022
+```
+
+  In this case:
+   - `alice` is the remote username
+   - `node` is the Teleport Node name
+   - `3022` is the remote SSH port, which is 3022 for Nodes running the Teleport
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:2250]:
```suggestion
   - `3022` is the remote SSH port, which is `3022` for Nodes running the Teleport
```

File: docs/pages/reference/cli.mdx [hunk 34]
```diff
@@ -2072,3 +2060,248 @@ $ tsh ls --search=staging,mac
...
+| `--data-dir`         | Directory to store internal bot data. In production environments access to this directory should be limited only to an isolated linux user as an owner with `0600` permissions. |
+| `--destination-dir`  | Directory to write short-lived machine certificates.                                           |
+| `--certificate-ttl`  | TTL of short-lived machine certificates.                                                       |
+| `--renewal-interval` | Interval at which short-lived certificates are renewed; must be less than the certificate TTL. |
+| `--join-method`      | Method to use to join the cluster. Can be `token` or `iam`.                                    |
+| `--oneshot`          | If set, quit after the first renewal.
+
+
+
+teleport start \
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:2301]: I think we should add this to a `code` fence with a `$` prompt.

**Comment by strideynet** [comment]
I do think if we are adding the `tbot` commands to the main CLI reference, we ought to remove the Machine ID CLI reference page so we aren't duplicating information in a way we need to modify it in two places (or maybe use a partial for it).

File: docs/pages/reference/cli.mdx (hunk 23 above)
Comment (ptgott) [reply docs/pages/reference/cli.mdx:22]: @rsyracuse Just checking to see if you wanted to incorporate this one

File: docs/pages/reference/cli.mdx [hunk 35]
```diff
@@ -53,8 +51,14 @@ For more information on subcommands when working with the `teleport` cli, use th
 
 ### teleport start
 
-The `teleport start` command includes a large number of optional configuration flags which change the behavior of the default command to support a wider spectrum of Teleport services.
-We recommend using the [Teleport configuration file](config.mdx) in all cases where possible.
+The `teleport start` command includes a large number of optional configuration flags.
+
+<Admonition title="Configuration Flags" type="tip">
+
+While configuration flags for `teleport start` can be used to set parameters for Teleport's configuration,
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:58]: I think we should either move the "The `teleport start` command..." sentence into this Admonition or turn this Admonition into body text, since I don't think the first sentence makes sense outside of the recommendation to use a configuration file. Otherwise, it just sounds like we're passing judgment on the number of optional configuration flags.

File: docs/pages/reference/cli.mdx [hunk 36]
```diff
@@ -2112,9 +2116,8 @@ $ tbot start \
    --auth-server=example.teleport.sh:443
 ```
 
-</ScopedBlock>
-
-<ScopedBlock scope={["oss","enterprise"]}>
+<Tabs>
+<TabItem scope={["cloud"]} label="Teleport Cloud">
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:2120]: Did you also add closing tags?

File: docs/pages/reference/cli.mdx [hunk 37]
```diff
@@ -1,53 +1,72 @@
...
 ---
 
-Teleport is made up of three CLI tools.
+Teleport is made up of four CLI tools.
 
-- [teleport](#teleport): The daemon that runs the Auth Service, Proxy Service, and other Teleport services.
-- [tsh](#tsh): A tool that lets end users interact with Teleport.
-- [tctl](#tctl): An administrative tool that can configure the Teleport Auth Service.
+- [teleport](#teleport): Supports the Teleport access plane by starting and configuring various Teleport services.
+- [tsh](#tsh): Allows end users to authenicate to Teleport and access resources in a cluster.
```
Comment (ptgott) [inline docs/pages/reference/cli.mdx:9]:
```suggestion
- [tsh](#tsh): Allows end users to authenticate to Teleport and access resources in a cluster.
```

**Review by ptgott — APPROVED** [review]
Approved, though there are still some minor outstanding comments
